apps/system/utils/Kconfig

```diff
@@ -81,6 +81,14 @@ config ENABLE_HEAPINFO
 	---help---
 		Show information about memory status per thread
 
+config ENABLE_IRQINFO
+	bool "irqinfo"
+	default y
```

Comment:
At beginning of new board, it can be needed. After that, I don't think this will be used.